service/controller/v16/version_bundle.go

```diff
@@ -16,7 +36,7 @@ func VersionBundle() versionbundle.Bundle {
 		Components: []versionbundle.Component{
 			{
 				Name:    "calico",
-				Version: "3.0.5",
+				Version: "3.2.3",
```

Comment:
Not sure how, we should keep this up to date.

Reply:
That can be the part of releasing operators story in our team.

Reply:
Go ahead and add a point.